.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
```

Comment:
- [ ] Kindly use the required linter file for this project. See [here](https://github.com/microverseinc/linters-config/tree/master/javascript) to get the correct linters.

Reply:
Hi - we did not used that linter because our repo includes HTML and CSS code, not only Javascript
![Screen Shot 2021-07-06 at 12 32 13 PM](https://user-images.githubusercontent.com/2223583/124636243-51735a00-de56-11eb-80a8-32d4ccf7b7af.png)